linkbrary/src/functions/useGetJson.js

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect, useCallback } from "react";
+
+const useGetJson = (func, param = null) => {
+  const [values, setValues] = useState([]);
+  const HandleLoad = useCallback(async () => {
+    let results;
+    try {
+      results = await func(param);
+    } catch (error) {
+      console.error(error);
+    }
+
+    if (!results) return;
+    setValues(results);
```

Comment:
```suggestion
    try {
      const results = await func(param);
      if (!results) return;
      setValues(results);
    } catch (error) {
      console.error(error);
    }


```
let을 안쓸수 있다면 최대한 안쓰는게 좋아요
코드가 길어지면 어디서 이 변수의 값이 바뀌는지 추적하기 어려워지거든요

그리고 앞으로는 값이 없을때 return하는 로직이 이 스코프를 끝내기 위함인지 아니면 undefined를 리턴하기 위함인지 생각해보시면서 사용하시면 좋을것 같아요.
기본적으로 return 만 사용하시면 return undefined를 의미하는거거든요. 좀더 명시적으로 빈값을 리턴하시려면 return null로 사용하는 경우도 많아요! 
https://helloworldjavascript.net/pages/160-null-undefined.html